lib/pages/main/main_vc.dart

```diff
@@ -180,202 +177,39 @@ class _MainVCState extends State<MainVC> {
-                                                  .size
-                                                  .width *
-                                              0.5,
-                                          child: CustomCard(
-                                            img:
-                                                'assets/img/ic_bite_report.png',
-                                            title: MyLocalizations.of(
-                                                context, 'report_biting_txt'),
-                                          ),
-                                        ),
-                                      ),
-                                      GestureDetector(
-                                        onTap: () {
-                                          loadingStream.add(true);
-                                          _createAdultReport();
-                                        },
-                                        child: Container(
-                                          width: MediaQuery.of(context)
-                                                  .size
-                                                  .width *
-                                              0.45,
-                                          height: MediaQuery.of(context)
-                                                  .size
-                                                  .width *
-                                              0.5,
-                                          child: CustomCard(
-                                            img:
-                                                'assets/img/ic_mosquito_report.png',
-                                            title: MyLocalizations.of(
-                                                context, 'report_adults_txt'),
-                                          ),
-                                        ),
-                                      ),
-                                    ],
+                                      Expanded(
+                                        flex: 2,
+                                        child: Column(
+                                          crossAxisAlignment: CrossAxisAlignment.start,
+                                          children: <Widget>[
+                                            Style.body(
+                                                MyLocalizations.of(context, 'what_to_do_txt'),
+                                                fontSize: 16),
+                                          ]),
+                                      )                                      
+                                    ]),
+                                  SizedBox(height: 25),
+                                  CustomCard(
+                                    text: 'single_mosquito',
+                                    image_path: 'assets/img/ic_mosquito_report.png',
+                                    color: '40DFD458',
```

Comment:
First two characters indicate the opacity (in this case, 40) and the rest of characters indicate the color